contracts/governance/BaseWeightedMultisig.sol

```diff
@@ -168,18 +168,13 @@ abstract contract BaseWeightedMultisig is IBaseWeightedMultisig {
      * @dev Updates the last rotation timestamp, and enforces the minimum rotation delay if specified
      */
     function _updateRotationTimestamp(bool enforceRotationDelay) internal {
-        uint256 lastRotationTimestamp_ = _baseWeightedMultisigStorage().lastRotationTimestamp;
-        uint256 currentTimestamp = block.timestamp;
-
-        if (enforceRotationDelay && (currentTimestamp - lastRotationTimestamp_) < minimumRotationDelay) {
-            revert InsufficientRotationDelay(
-                minimumRotationDelay,
-                lastRotationTimestamp_,
-                currentTimestamp - lastRotationTimestamp_
-            );
+        uint256 timeSinceRotation_ = timeSinceRotation();
+
+        if (enforceRotationDelay && timeSinceRotation_ < minimumRotationDelay) {
+            revert InsufficientRotationDelay(minimumRotationDelay, lastRotationTimestamp(), timeSinceRotation_);
         }
 
-        _baseWeightedMultisigStorage().lastRotationTimestamp = currentTimestamp;
+        _baseWeightedMultisigStorage().lastRotationTimestamp = block.timestamp;
```

Comment:
What's the gas cost difference for rotateSigners before and after this change?

Reply:
Yeah, this is why I hadn't reused it. Code is still readable so I prefer to keep it as is